Tabloid/appsettings.json

```diff
@@ -9,5 +9,6 @@
   "AllowedHosts": "*",
   "ConnectionStrings": {
     "DefaultConnection": "server=localhost\\SQLExpress;database=Tabloid;integrated security=true"
-  }
+  },
+  "FirebaseProjectId": "tabloid-4bf15"
```

Comment:
This line should be in appsettings.Local.json, not here.

Reply:
Fixed!